torch/autograd/gradcheck.py

```diff
@@ -1455,7 +1526,8 @@ def gradcheck(
             implemented for R to R functions. If none of the inputs and outputs are complex
             a faster implementation of gradcheck that no longer computes the entire jacobian
             is run; otherwise, we fall back to the slow implementation.
-
+        masked (bool, optional): if True, the gradients of unspecified elements of
+            sparse tensors are masked out (default, False).
```

Comment:
nit: I would say "are ignored". Also meta question, do we have a doc that discuss the sparse vs masked semantics? If so, we can point it from there.

Reply:
Fixed.
I believe we don't have such documentation that could be referenced here. This is partly because our aim has been to move all masked-semantics-related features under MaskedTensor and therefore discussing masked semantics in connection to sparse tensors in official documentation has become kind of taboo.

Reply:
I think linked to the MaskedTensor page sounds good then!